src/Command/AbstractCommand.php

```diff
@@ -75,26 +77,54 @@ protected function getTableCount(string $entityName): int
     /**
      * Return an array of entity-metadata for all entities
      * that have at least one encrypted property.
+     * The returned array also contains counts of the total
+     * amount of encrypted properties and the count of
+     * encrypted properties per entity which includes 0 counts.
      */
-    protected function getEncryptionableEntityMetaData(): array
+    protected function getEncryptionableEntityDetails(): array
```

Comment:
I think this could use a docbloc helping PHPStan and users on what to expect in the result of this method:

```php
/**
 * @return array{
 *     array<ClassMetadata>,
 *     propertyCountPerEntity: array<class-string, int>,
 *     totalPropertyCount: int,
 * }
 */
```